### DISCUSSION TITLE
FLV格式已支持4K

### DISCUSSION THREAD

**Discussion by BakaJzon** [the1812/Bilibili-Evolved#1308] [功能建议 / Ideas]
PlayURL：http://api.bilibili.com/x/player/playurl?bvid=BV1fK4y1t7hj&cid=196018899&fourk=1&qn=120&fnver=0&fnval=0
视频：http://b23.tv/BV1fK4y1t7hj 
标题：【4K 120FPS】你的设备顶得住吗？4K120帧技术演示片
4K FLV 视频：http://cn-jxnc-cmcc-bcache-12.bilivideo.com/upgcxcode/99/88/196018899/196018899-1-120.flv?e=ig8euxZM2rNcNb41hwdVhwdlnwRVhwdVhoNvNC8BqJIzNbfqXBvEqxTEto8BTrNvN0GvT90W5JZMkX_YN0MvXg8gNEV4NC8xNEV4N03eN0B5tZlqNxTEto8BTrNvNeZVuJ10Kj_g2UB02J0mN0B5tZlqNCNEto8BTrNvNC7MTX502C8f2jmMQJ6mqF2fka1mqx6gqj0eN0B599M=&uipk=5&nbs=1&deadline=1607774597&gen=playurl&os=bcache&oi=1974139840&trid=cc6f99703418445ea09cd42642b06bfbu&platform=pc&upsig=21cd206c8d71ac70c717e62e57757eff&uparams=e,uipk,nbs,deadline,gen,os,oi,trid,platform&cdnid=6639&mid=00000000&orderid=0,3&agrr=1&logo=80000000

看来B站只是没有更新FLVPlayer，但使用时最好小心点
希望能添加4K FLV 下载选项，DASH太不方便了，蟹蟹٩('ω')و

**Comment by the1812** [comment]
下次更新正式版就会支持

**Reply by BakaJzon** [reply]
emmm
![image](https://user-images.githubusercontent.com/55239052/103410232-6d472c00-4ba5-11eb-8506-4b4c98bca39b.png)

**Reply by the1812** [reply]
现在只有预览版支持
